eksctl/Makefile

```diff
@@ -18,7 +18,9 @@ check:
 .PHONY: test
 test:
 	@make check;
-	docker-compose -f docker-compose.test.yml run --rm sut;
+	docker-compose -f docker-compose.test.yml up --no-start sut
+	docker cp $(shell pwd)/goss `basename $$PWD`:/goss
```

Comment:
Why do you change `mount` to `docker cp` ?

Reply:
@cw-sakamoto 
In CircleCI DLC, the host that runs docker cli is different from the host that runs docker daemon.
Therefore, volume mount is not possible.

Use `docker cp` because `/goss` directory is necessary for test execution.